src/Utility/Examiner.php

```diff
@@ -36,7 +36,16 @@ class Examiner {
     // Compound.
     'http://vocab.getty.edu/aat/300242735' => [],
     // Digital Document.
-    'https://schema.org/DigitalDocument' => [],
+    'https://schema.org/DigitalDocument' => [
+      [
+        'bundle' => 'extracted_text',
+        'use_uri' => 'http://pcdm.org/use#ExtractedText',
+      ],
+      [
+        'bundle' => 'image',
+        'use_uri' => 'http://pcdm.org/use#ServiceFile',
```

Comment:
Also, pretty sure this should've been dealing "thumbnails", not "service files":
- https://github.com/discoverygarden/dgi-starter/blob/d481c6d8b9a993d1df0848e07d2529433764c106/config/sync/context.context.pdf_original_file.yml#L36
- https://github.com/discoverygarden/dgi-starter/blob/d481c6d8b9a993d1df0848e07d2529433764c106/config/sync/system.action.digital_document_generate_a_thumbnail_from_an_original_file.yml#L18